src/reducers/accounts.js

```diff
@@ -125,6 +125,7 @@ export const addAccounts = accounts => async dispatch => {
       return {
         proxyRole: otherRole,
         address: linkedAddress,
+        // type: 'browser',
```

Comment:
3 - without adding the account, we don't have access the type, and the app will crash.